tools/convert/configData.yaml

```diff
@@ -880,7 +897,7 @@ groups:
       Controls the field names to use for the event ID fields. These fields are
       used to identify events that are part of the same trace.
     fields:
-      - name: TraceIDFieldNames
+      - name: Trace
```

Comment:
I did this because it stuttered -- the category it's in is "IDFieldNames" so the value was "IDFieldNames.TraceIDFieldNames". It's the same pattern I've followed elsewhere with things like APIHost.

But I've adjusted it, and hopefully you'll be happy with it:
```
IDFields:
    TraceNames
    ParentNames
```

Reply:
Perfect